src/status_im/currency/core.cljs

```diff
@@ -14,6 +14,6 @@
   (rf/merge cofx
             (multiaccounts.update/multiaccount-update
              :currency
-             currency
+             "USD"
```

Comment:
btw, we need to test this with lower case currency too - did this affect anything? otherwise we need to update how we store currency in DB